src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -1097,4 +1098,32 @@ private static Date toDate(LocalDate localDate) {
 		return Optional.ofNullable(localDate).map(t -> Date.from(t.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant())).orElse(null);
 	}
 
-}
+	public JasperReportResultDto getInventoryReportPdf(MedicalInventory inventory, String jasperFileName, int printQtyReal) throws OHServiceException {
+		try {
+			Hospital hosp = hospitalManager.getHospital();
```

Comment:
This line isn't needed.  the `getHospitalParameters()` call in line 1105 get the hospital object isself.